app/mailers/event_mailer.rb

```diff
@@ -0,0 +1,10 @@
+class EventMailer < ApplicationMailer
+  def cancelled(params)
```

Comment:
Can we just send the `event` object here as there is nothing else? `params` looks like a lot :)